packages/proof/src/get-snark-artifacts.browser.ts

```diff
@@ -0,0 +1,9 @@
+/* istanbul ignore file */
+import { SnarkArtifacts } from "./types"
+
+export default async function getSnarkArtifacts(treeDepth: number): Promise<SnarkArtifacts> {
+    return {
+        wasmFilePath: `https://semaphore.cedoor.dev/artifacts/${treeDepth}/semaphore.wasm`,
```

Comment:
nit: switch the host from your personal @cedoor to PSE one?

Reply:
This is temporary. We'll change host as soon as we run the ceremony and have the actual artifacts.